iroha_p2p/src/network.rs

```diff
@@ -250,14 +270,14 @@ pub struct GetConnectedPeers;
 #[derive(Clone, Debug, iroha_actor::Message)]
 pub struct ConnectedPeers {
     /// Connected peers' ids
-    pub peers: HashSet<PeerId>,
+    pub peers: HashSet<PublicKey>,
 }
 
 /// Variants of messages from [`Peer`] - connection state changes and data messages
 #[derive(Clone, Debug, iroha_actor::Message, Decode)]
 pub enum PeerMessage<T: Encode + Decode> {
     /// Peer just connected and finished handshake
-    Connected(PeerId),
+    Connected(PeerId, u64),
```

Comment:
Can you add a type alias `ConnectionId` to u64? So that it is easier to understand what params this tuple has.

Reply:
Added.